src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/TasksController.kt

```diff
@@ -134,30 +134,26 @@ class TasksController(
 
   override fun tasksGet(
     type: TaskType?,
+    types: List<TaskType>?,
```

Comment:
This looks great, thks. I think there's a tiny typo in the commit message, which for clarity might be worth fixing up. I think you're meaning to say:

> Previously the caller could get tasks of all types, **or** of a specific type